feature/bgp/static_route_bgp_redistribution/otg_tests/static_route_bgp_redistribution_test/static_route_bgp_redistribution_test.go

```diff
@@ -206,13 +209,13 @@ func configureDUTStatic(t *testing.T, dut *ondatra.DUTDevice) {
 		attachTagSetToStaticRoute(t, dut, "2024:db8:128:128::/64", "tag-static-v6")
 	}
 
-	ipv6StaticRouteNextHop := ipv6StaticRoute.GetOrCreateNextHop("1")
+	ipv6StaticRouteNextHop := ipv6StaticRoute.GetOrCreateNextHop("0")
```

Comment:
this seems like a super arbitrary change w/o explanation.

does your implementation require the next hops to start with the zero index? OpenConfig does not impose such restrictions.

Reply:
this change is not necessary, its just that incrementing index is of no use.

Reply:
Following precedent, we've been configuring from a "0" based index, so I am ok with this change.  Agreed though, one could start with "1" (as in the current code) and it should still work.


Example precedent:
https://github.com/openconfig/featureprofiles/blob/75128ea737b39431dd948df1676a3f60ddc40add/feature/staticroute/otg_tests/basic_static_route_support_test/basic_static_route_support_test.go#L510-L513